### DISCUSSION TITLE
Link re-order

### DISCUSSION THREAD

**Discussion by jam3sperkins** [JulianPrieber/littlelink-custom#119] [Feature requests]
Hi there,

Loving the product!

I love the idea of having a drag-and-drop way of re-ordering links. 

Currently, I have to manually go into the links and adjust the numbering to prioritise the latest link to the top.

If I set something at 0, 1, 2, 3, 4 etc and then pin the link at position 0, it drops it to the bottom of my links page (not sure if this is a bug or not).

But if something like sortablejs was used on the links page, you could easily drag to re-order based on you're preference rather than entering into each link manually.

I was also unable to find in the documentation what the "order" and "pin" options do. So I was not sure if the order starts from 0 or 1.

Thanks,
James

**Comment by JulianPrieber** [comment]
**Hi James,**

This is already on our list of things to add in the future (see https://github.com/JulianPrieber/littlelink-custom/discussions/49).

Sadly, as of now, I can't really implement this because I don't know much about JavaScript. This is a passion project and my first ever open source repository of this scale. I'm not a programmer, I'm self-taught in everything I do here and still fairly new to this.

I would love to add this as soon as I feel more comfortable working with JS.

Until then, anyone is welcome to add this on their own, and I would be more than happy to help wherever I can.

<br>

As for your problem with the pinning of links, as far as I can tell this was supposed to bring the buttons to the top. I wasn't the one who implemented this, and I'm not really sure how this works exactly. I'm working on a fix and will hopefully be able to implement this in a future update.

All the best,
Julian

**Comment by JulianPrieber** [comment]
@jam3sperkins 

We actually managed to work out a solution that implements this. We're doing a public beta test on: https://demo.littlelink-custom.com

**Reply by jam3sperkins** [reply]
Look at that! That works a charm. Awesome stuff, I like it! Thank you so much for looking into it.
